src/imp/openssl.rs

```diff
@@ -112,6 +112,18 @@ impl<S> From<ErrorStack> for HandshakeError<S> {
 fn ctx() -> Result<SslContext, Error> {
     let mut ctx = try!(SslContext::new(SslMethod::Sslv23));
     try!(ctx.set_default_verify_paths());
+
+    // options to enable and cipher list lifted from libcurl
+    let mut opts = ssl::SSL_OP_ALL;
+    opts |= ssl::SSL_OP_NO_TICKET;
+    opts |= ssl::SSL_OP_NO_COMPRESSION;
+    opts &= !ssl::SSL_OP_NETSCAPE_REUSE_CIPHER_CHANGE_BUG;
+    opts &= !ssl::SSL_OP_DONT_INSERT_EMPTY_FRAGMENTS;
+    opts |= ssl::SSL_OP_NO_SSLV2;
+    opts |= ssl::SSL_OP_NO_SSLV3;
```

Comment:
I think that's a good point yeah. I've pushed a commit to disable it on Windows, but calling [`set_protocol_version_min`](http://sfackler.github.io/rust-security-framework/doc/v0.1.8/security_framework/secure_transport/struct.SslContext.html#method.set_protocol_version_min) on OSX will require some more finesse because it's only defined on 10.8+